doc/src/learn/sponsored-transactions.md

```diff
@@ -11,16 +11,6 @@ You can use Sui Sponsored transactions to:
  * Sponsor transactions you initiate as the sponsor.
  * Provide a wildcard GasData object to users. The object covers the gas fees for a user transaction. The GasData object covers any fee amount determined for the transaction as long as the budget is sufficient.
 
-## Considerations for Sponsored Transactions
```

Comment:
Should we mention the usage of `Argument::GasCoin`? They might want to guard against it's usage

Reply:
Hi @longbowlu Prefer to add additional info here, or merge this PR and make an additional change for @tnowacki request?